spec/features/multiple_emails/reset_password_spec.rb

```diff
@@ -41,7 +41,7 @@
 
     mail = double
     expect(mail).to receive(:deliver_later)
-    expect(UserMailer).to receive(:email_confirmation_instructions).with(
+    expect(UserMailer).to receive(:unconfirmed_email_instructions).with(
```

Comment:
should we change the description of this spec?
`it sends a create an account email` to `it sends the unconfirmed address create account email` or something?